app/src/main/java/org/eyeseetea/malariacare/domain/entity/Country.java

```diff
@@ -29,46 +22,26 @@ public long getId() {
         return id;
     }
 
-    public void setId(long id) {
+    private void setId(long id) {
         this.id = id;
     }
 
     public String getUid() {
         return uid;
     }
 
-    public void setUid(String uid) {
+    private void setUid(String uid) {
         this.uid = uid;
     }
 
     public String getName() {
         return name;
     }
 
-    public void setName(String name) {
+    private void setName(String name) {
         this.name = name;
     }
 
-    @Override
-    public boolean equals(Object o) {
```

Comment:
Why are you remove equals and hashcode methods? These methods can be useful in many comparison situations. If there is no reason, create them again, please.

Reply:
I'm aware of the value that the equal method brings. I removed it because for the moment it hasn't been used it anywhere and one of the recommendations that I found on the internet for eliminating anemic entities was removing not used methods.

I will bring it back!